prov/efa/test/README.md

```diff
@@ -0,0 +1,53 @@
+# EFA unit tests
+
+## How to run
+
+To run efa unit tests, you will need to have cmocka installed.
+See: https://github.com/clibs/cmocka/blob/master/INSTALL.md
+
+You will need to configure libfabric with ```--with-cmocka <path_to_cmocka_install>```.
```

Comment:
Can you do ```--with-cmocka=<path_to_cmocka_install>``` to keep format consistent with example

Reply:
done